### PR TITLE
Consult default.d/local.d for ini files

### PR DESCRIPTION
**Pull request by rnewson**
COUCHDB-3089

### CODE REVIEW

**Comment by kxepal** [comment]
+1

File: src/config_app.erl [hunk 1]
```diff
@@ -48,5 +49,18 @@ command_line() ->
...
+        filename:join(Etc, "default.d"),
+        filename:join(Etc, "local.ini"),
+        filename:join(Etc, "local.d")
+    ],
     lists:filter(fun filelib:is_file/1, Default).
+
+expand_dirs(File) ->
+    case filelib:is_dir(File) of
+        true ->
+            filelib:wildcard(File ++ "/*.ini");
```
Comment (davisp) [inline src/config_app.erl]: You should add a sort here so that the order they're applied is defined.

**Comment by davisp** [comment]
+1 after adding the sort

File: src/config_app.erl (hunk 1 above)
Comment (rnewson) [inline src/config_app.erl]: the current code does a glob though, and no sort, so that would be a change, albeit a good one. still want the short?

**Pull request merged**